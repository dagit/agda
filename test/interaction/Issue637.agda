module Issue637 where
