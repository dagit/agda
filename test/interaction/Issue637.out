(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Error*" "stack overflow" nil)
(agda2-status-action "")
Stack space overflow: current size 8 bytes.
Use `+RTS -Ksize -RTS' to increase it.
